.github/workflows/playwright.yml

```diff
@@ -0,0 +1,21 @@
+name: Playwright E2E Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  e2e-tests:
+    runs-on: ubuntu-latest
```

Comment:
I think we should bind this to one of the ubuntu releases rather than depending on the latest release. Like `ubuntu:jammy`